.gitignore

```diff
@@ -1,9 +1,6 @@
-node_modules/
-.sublime-gulp.cache
-**/*.sublime-project
-**/*.sublime-workspace
-**/*.sass-cache
-workspace
-demo/components.js
-demo/components.js.map
-*.log*
+node_modules
+.coverage
+.cache
+/.DS_Store
+/.env.*
```

Comment:
@giotramu remove the leading  `/` otherwise git will ignore only directories matching that expression

Reply:
Done.